public/assets/manifest-863d1fc0a1a5a48ae2899ebf34959a29.json

```diff
@@ -1 +1 @@
3489.png":{"logical_path":"flags/Palestine.png","mtime":"2016-06-12T16:08:38-04:00","size":35007,"digest":"78e36a0594cb41e3eae6afbf77b03489"},"flags/Paraguay-0cae46376ed6488080454c4b77fe84f9.png":{"logical_path":"flags/Paraguay.png","mtime":"2016-06-12T16:08:38-04:00","size":47725,"digest":"0cae46376ed6488080454c4b77fe84f9"},"flags/Philippines-dd529dd6f0f383627c9dac2829112ada.png":{"logical_path":"flags/Philippines.png","mtime":"2016-06-12T16:08:38-04:00","size":51339,"digest":"dd529dd6f0f383627c9dac2829112ada"},"flags/Poland-aae6f5e000ccff9b6b9cc334e9cf0aa8.png":{"logical_path":"flags/Poland.png","mtime":"2016-06-12T16:08:38-04:00","size":34718,"digest":"aae6f5e000ccff9b6b9cc334e9cf0aa8"},"flags/Portugal-010aadcd44c6f41b54321c8ed19be8d8.png":{"logical_path":"flags/Portugal.png","mtime":"2016-06-12T16:08:38-04:00","size":67594,"digest":"010aadcd44c6f41b54321c8ed19be8d8"},"flags/Quatar-4029f80901066709dc0eafe0e9369d23.png":{"logical_path":"flags/Quatar.png","mtime":"2016-06-12T16:08:38-04:00","size":53931,"digest":"4029f80901066709dc0eafe0e9369d23"},"flags/Romainia-8c047836a37499e0ace5390bc1d83f8d.png":{"logical_path":"flags/Romainia.png","mtime":"2016-06-12T16:08:38-04:00","size":46637,"digest":"8c047836a37499e0ace5390bc1d83f8d"},"flags/Russia-0612efad5db275d60107407d3fdefc75.png":{"logical_path":"flags/Russia.png","mtime":"2016-06-12T16:08:38-04:00","size":42303,"digest":"0612efad5db275d60107407d3fdefc75"},"flags/Russian Federation-0612efad5db275d60107407d3fdefc75.png":{"logical_path":"flags/Russian Federation.png","mtime":"2016-06-08T22:56:39-04:00","size":42303,"digest":"0612efad5db275d60107407d3fdefc75"},"flags/SanMarino-388ea36267f1621f07067cce7f602a7c.png":{"logical_path":"flags/SanMarino.png","mtime":"2016-06-12T16:08:38-04:00","size":53631,"digest":"388ea36267f1621f07067cce7f602a7c"},"flags/SaudiArabia-4147b22d93babe9814719a5ad6df5733.png":{"logical_path":"flags/SaudiArabia.png","mtime":"2016-06-12T16:08:38-04:00","size":61741,"digest":"4147b22d93babe9814719a5ad6df5733"},"flags/Serbia-72137f257f535a54dce2b3ddb64663dc.png":{"logical_path":"flags/Serbia.png","mtime":"2016-06-12T16:08:38-04:00","size":63947,"digest":"72137f257f535a54dce2b3ddb64663dc"},"flags/Singapore-9a050af0253475debe68879e2ce6e860.png":{"logical_path":"flags/Singapore.png","mtime":"2016-06-12T16:08:38-04:00","size":40858,"digest":"9a050af0253475debe68879e2ce6e860"},"flags/Slovakia-9e8b319bbf0c08fb3dd8d917863ac76b.png":{"logical_path":"flags/Slovakia.png","mtime":"2016-06-12T16:08:38-04:00","size":45234,"digest":"9e8b319bbf0c08fb3dd8d917863ac76b"},"flags/Slovenia-0b94136786cc582de22295355809d5b0.png":{"logical_path":"flags/Slovenia.png","mtime":"2016-06-12T16:08:38-04:00","size":44816,"digest":"0b94136786cc582de22295355809d5b0"},"flags/SouthAfrica-18d08881f3ad10e837c5bed14276978c.png":{"logical_path":"flags/SouthAfrica.png","mtime":"2016-06-12T16:08:38-04:00","size":50954,"digest":"18d08881f3ad10e837c5bed14276978c"},"flags/SouthKorea-d0e6d667aa5f5b5c409a1b4f61382206.png":{
```

Comment:
Mm, yes?  I would like to keep track but really only because I haven't looked too closely at how heroku does compiling.